web/src/main/webapp/WEB-INF/jsp/member/create.jsp

```diff
@@ -0,0 +1,22 @@
+<%@ page language="java" contentType="text/html; charset=UTF-8" pageEncoding="UTF-8"%>
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core"%>
+<%@ taglib prefix="form" uri="http://www.springframework.org/tags/form"%>
+<%@ taglib tagdir="/WEB-INF/tags" prefix="x"%>
+<x:base title="Register new Member">
```

Comment:
I'd use different title. This page is supposed to be used by a user to register himself, not someone else. So something like "Sing up" or "Register"